### ISSUE TITLE
About training setting of Trades

### ISSUE THREAD

**Issue by jiaxiaojunQAQ** [P2333/Bag-of-Tricks-for-AT#3]
How much epoch is used in Trades in this paper ?  And if the learning rate is the same as the original Trades?

**Comment by P2333** [comment]
The TRADES experiments in our paper are run for 110 epochs, and the learning rate decays by a factor of 10 at 100th and 105th epochs, respectively.
